modules/javafx.graphics/src/main/java/javafx/scene/text/Text.java

```diff
@@ -1267,6 +1269,42 @@ private boolean doComputeContains(double localX, double localY) {
         return TransformedShape.translatedShape(shape, x, y);
     }
 
+    /**
+     * The size of a tab stop in spaces.
+     *
+     * @return the {@code tabSize} property
+     *
+     * @defaultValue {@code 8}
+     *
+     * @since 14
+     */
+    public final IntegerProperty tabSizeProperty() {
+        return getTextAttribute().tabSizeProperty();
+    }
+
+    /**
+     * Gets the size of a tab stop in spaces.
+     * @return the size of a tab in spaces
+     * @since 14
+     */
+    public final int getTabSize() {
+        if (attributes == null || attributes.tabSize == null) {
+            return TextLayout.DEFAULT_TAB_SIZE;
+        }
+        return getTextAttribute().getTabSize();
+    }
+
+    /**
+     * Sets the size of a tab stop.
+     * @param spaces the size of a tab in spaces. Defaults to 8.
+     * Minimum is 1, lower values will be clamped to 1.
+     * @since 14
+     */
```

Comment:
Same comment as on the set method. You can move the comment about values being clamped to 1 to the property method (which will then be the only method with javadoc comments). If we use my recommendation of clamping on usage, I recommend the following language, borrowed from `Node::opacity`:

```
Values less than 1 are treated as 1.
```